.gitignore

```diff
@@ -19,3 +19,4 @@ target/
 
 # Still not sure why cargo creates this in the root directory
 /elf
+**/elf/*
```

Comment:
Why not replace the previous line and this line with `elf/` ?

Reply:
this doesn't appear to have changed, so, same question

Reply:
Changed to `elf/`